### ISSUE TITLE
Introduce a new category for AI / ML learning tools supporting FMI?

### ISSUE THREAD

**Issue by chrbertsch** [traversaro/awesome-fmi#31]
More and more AI and Machine Learning tools support FMI. 
- FMIflux https://github.com/ThummeTo/FMIFlux.jl
- JuliaSim https://help.juliahub.com/JuliaSim/Tutorials/FMUs/
- bons.ai
-- https://github.com/microsoft/FMU-bonsai-connector
-- https://docs.microsoft.com/en-us/bonsai/guides/add-simulator?tabs=add-cli%2Ctrain-inkling&pivots=sim-platform-fmu
- openAI gyms  (various)
--  https://github.com/ucuapps/modelicagym
-- https://pypi.org/project/openmodelica-microgrid-gym/
-- fmpy, see e.g. https://github.com/CATIA-Systems/FMPy/issues/277

Perhaps it would be good to introduce a new category for them?

**Comment by traversaro** [comment]
Good idea, it could be a new category after "Tools"! If you have a precise idea on where to place this on the existing document, feel free to propose it, thanks!
